test/libsolidity/util/SoltestTypes.h

```diff
@@ -229,6 +230,12 @@ struct FunctionCallArgs
 	}
 };
 
+enum class FunctionCallValueCoin
```

Comment:
"denomination" instead of "coin"? Or "unit"?

Reply:
Fixed